scripts/update-nodejs-versions.js

```diff
@@ -70,7 +71,11 @@ async function getNodeJsVersion(version) {
 async function main() {
   const versions = await getNodeJsVersions();
   const nodeRepositories = await Promise.all(versions.map(getNodeJsVersion));
-
+  console.log('""" Generated code; do not edit');
+  console.log('Update by running yarn update-nodejs-versions\n"""\n');
+  // Suppress buildifier
+  console.log('# @unsorted-dict-items');
+  console.log('NODE_VERSIONS = {');
```

Comment:
nit: perhaps add to a single array and join, rather than logging multiple times? Kinda preference though :)